hip-service/Link/Patient/PatientVerification.cs

```diff
@@ -1,26 +1,76 @@
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
 using System.Threading.Tasks;
 using hip_service.OTP;
 using HipLibrary.Patient.Model.Response;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using Optional;
 
 namespace hip_service.Link.Patient
 {
     public class PatientVerification: IPatientVerification
     {
-        private readonly OtpVerification otpVerification;
-
-        public PatientVerification(OtpVerification otpVerification)
+        private readonly IConfiguration configuration;
+        
+        public PatientVerification(IConfiguration configuration)
         {
-            this.otpVerification = otpVerification;
+            this.configuration = configuration;
         }
-
-        public async Task<Error?> SendTokenFor(Session session)
+        
+        public async Task<OtpMessage> SendTokenFor(Session session)
         {
-            return await otpVerification.GenerateOtp(session);
+            var uri = new Uri(configuration.GetConnectionString("OTPGenerationConnection"));
+            var verifyMessage = await PostCallToOTPServer(uri
+                , CreateHttpContent(session));
+            return verifyMessage.ValueOr((OtpMessage) null);
         }
         
-        public async Task<Error?> Verify(string sessionId, string value)
+        public async Task<OtpMessage> Verify(string sessionId, string value)
         {
-            return await otpVerification.CheckOtpValue(sessionId, value);
+            var uri = new Uri(configuration.GetConnectionString("OTPVerificationConnection"));
+            var verifyOtpRequest = new OtpVerificationRequest(sessionId, value);
+            var verifyMessage = await PostCallToOTPServer(uri
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4807294)